app/models/image.rb

```diff
@@ -1,6 +1,7 @@
 class Image < ApplicationRecord
   belongs_to :image_data
   belongs_to :edition
+  has_one :edition_lead_image, dependent: :destroy
```

Comment:
If you did agree EditionLeadImage has one Image, then the Image would belong to the EditionLeadImage